torchao/experimental/kernels/mps/src/dispatch.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <functional>
+
+namespace torchao {
+namespace kernels {
+namespace mps {
+namespace lowbit {
+namespace dispatch {
```

Comment:
Nit (why not use C++17 conventions)
```suggestion
namespace torchao::kernels::mps::lowbit::dispatch {
```